.github/workflows/ci.yml

```diff
@@ -1,30 +1,131 @@
----
-
 name: CI
 
 on:
   pull_request:
-    branches: [devel]
+    branches:
+      - devel
 
   push:
-    branches: [devel]
+    branches:
+      - devel
 
 jobs:
+  sanity:
+    name: ${{ matrix.test.name }}
+    runs-on: ubuntu-20.04
+    container:
+      image: quay.io/ansible/ansible-builder-test-container:1.0.0
+      env:
+        PIP_CACHE_DIR: ${{ runner.temp }}/.cache/pip
+
+    strategy:
+      fail-fast: false
+      matrix:
+        test:
+          - name: Lint
+            env: linters
+
+          - name: Docs
+            env: docs
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup test environment
+        run: tox -e ${{ matrix.test.env }} --notest
```

Comment:
Instead of `-e` in each place, consider just adding
```yaml
env:
    TOXENV: ${{ matrix.env }}
```
to this job. It's a widely used practice across the ecosystem.

Reply:
That's a good idea.